gradle/validation/owasp-dependency-check/exclusions.xml

```diff
@@ -23,43 +23,32 @@
 
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
   <suppress>
-    <notes><![CDATA[
-   file name: simple-xml-safe-2.7.1.jar
-   We use a safe fork that fixes this
-   ]]></notes>
+    <notes><![CDATA[simple-xml-safe is a safe xml-safe fork]]></notes>
     <packageUrl regex="true">^pkg:maven/com\.carrotsearch\.thirdparty/simple\-xml\-safe@.*$</packageUrl>
     <cve>CVE-2017-1000190</cve>
   </suppress>
   <suppress>
-    <notes><![CDATA[
-   file name: netty-transport-native-epoll-4.1.29.Final.jar
-   We only use netty as a client towards Zookeeper
-   ]]></notes>
-    <packageUrl regex="true">^pkg:maven/io\.netty/netty\-.*@.*$</packageUrl>
-    <cpe>cpe:/a:netty:netty</cpe>
+    <notes><![CDATA[apache-mime4j has different releases than apache james]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.james/apache\-mime4j.*@.*$</packageUrl>
+    <cve>CVE-2021-38542</cve>
+    <cve>CVE-2021-40110</cve>
+    <cve>CVE-2021-40111</cve>
+    <cve>CVE-2021-40525</cve>
   </suppress>
   <suppress>
-    <notes><![CDATA[
-   file name: dirgra-0.3.jar
-   We will sandbox JRuby
-   ]]></notes>
-    <packageUrl regex="true">^pkg:maven/org\.jruby/dirgra@.*$</packageUrl>
-    <cpe>cpe:/a:jruby:jruby</cpe>
+    <notes><![CDATA[apache-rat-tasks is not tasks]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.rat/apache\-rat\-tasks@.*$</packageUrl>
+    <cpe>cpe:/a:tasks:tasks</cpe>
   </suppress>
   <suppress>
-    <notes><![CDATA[
-   file name: carrot2-guava-18.0.jar
-   Only used with clustering engine, and the risk is DOS attack
-   ]]></notes>
-    <packageUrl regex="true">^pkg:maven/org\.carrot2\.shaded/carrot2\-guava@.*$</packageUrl>
-    <cpe>cpe:/a:google:guava</cpe>
+    <notes><![CDATA[zookeeper is not Jetty]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.zookeeper/zookeeper.*@.*$</packageUrl>
+    <cve>CVE-2021-28164</cve>
+    <cve>CVE-2021-34429</cve>
```

Comment:
Zookeeper is being flagged for these CVEs for no good reason. We upgrade Jetty and these are Jetty CVEs - they can't be fixed from Zookeeper itself. So suppress these misflagged CVEs specifically - its impossible to do via a cpe suppression.